indra/newview/scripts/lua/util.lua

```diff
@@ -41,4 +41,11 @@ function util.equal(t1, t2)
     return util.empty(temp)
 end
 
+function util.check_response(res)
+  if res.error then
+    error(res.error)
+  end
+  return res
+end
```

Comment:
It's a good idea to centralize that logic. I decided to [bake it into `leap.lua`](https://github.com/secondlife/viewer/pull/1555/files#diff-4edaba8867aa1260f8c65602f02cb56cd59d5114829d78faa133d11a98e1af14), which would mean `UI.lua` can just call `leap.request()`. Should we cherry-pick that `leap.lua` commit?

Reply:
Sounds good, I'll cherry-pick those changes.